cli/Dockerfile

```diff
@@ -28,7 +28,8 @@ RUN apk --update --no-cache add \
       openssh-client \
       patch \
       git \
-      jq
+      jq \
+      less
```

Comment:
can we have this in alphabetically order please :pray:

Reply:
no

Reply:
I was going to and then saw the rest weren't so 🤷